Tests/AK/TestDuration.cpp

```diff
@@ -328,6 +328,43 @@ BENCHMARK_CASE(years_to_days_since_epoch_benchmark)
     }
 }
 
+TEST_CASE(days_since_epoch)
+{
+    EXPECT_EQ(days_since_epoch(1970, 1, 1), 0);
+    EXPECT_EQ(days_since_epoch(1970, 1, 2), 1);
+    EXPECT_EQ(days_since_epoch(1970, 2, 1), 31);
+    EXPECT_EQ(days_since_epoch(1970, 2, 27), 57);
+    EXPECT_EQ(days_since_epoch(1970, 2, 28), 58);
+    EXPECT_EQ(days_since_epoch(1970, 2, 29), 59); // doesn't really exist
+    EXPECT_EQ(days_since_epoch(1970, 3, 1), 59);
+    EXPECT_EQ(days_since_epoch(1971, 1, 1), 365);
+    EXPECT_EQ(days_since_epoch(1972, 1, 1), 730);
+    EXPECT_EQ(days_since_epoch(1972, 2, 1), 761);
+    EXPECT_EQ(days_since_epoch(1972, 2, 27), 787);
+    EXPECT_EQ(days_since_epoch(1972, 2, 28), 788);
+    EXPECT_EQ(days_since_epoch(1972, 2, 29), 789);
+    EXPECT_EQ(days_since_epoch(1972, 3, 1), 790);
+
+    // At least shouldn't crash:
+    EXPECT_EQ(days_since_epoch(1971, 1, 0), 364);
+    // FIXME shouldn't crash: EXPECT_EQ(days_since_epoch(1971, 0, 1), 334);
+    // FIXME shouldn't crash: EXPECT_EQ(days_since_epoch(1971, 0, 0), 333);
+    // FIXME shouldn't crash: EXPECT_EQ(days_since_epoch(1971, 13, 3), ???);
+
+    // I can't easily verify that these values are perfectly exact and correct, but they're close enough.
```

Comment:
(1) Unix time is always Gregorian even before that calendar was really introduced, so we don't have to observe the Julian/Gregorian calendar discontinuity in 1500-something.
(2) Unix time doesn't know that year 0 doesn't exist, so all negative years are shifted by 1
(3) Unix time doesn't know that leap year rules change before the Gregorian/Julian change

... and the list goes on. This API should really be removed at some point since it's hopelessly naive without precise timezone and calendar information, I only left it in to not overburden the initial PR. We should really be using special parsers depending on where the data comes from and what calendar they define; for instance the Kernel usage is limited to the CD-ROM definition of a Gregorian/Julian calendar with Unix timezone.